.github/workflows/nightly_macos_apple_silicon.yml

```diff
@@ -29,7 +29,7 @@ jobs:
         env:
             DATE: ${{ env.DATE }}
             SHA: ${{ env.SHA }}
-        run: echo "RELEASE_TAR_FILENAME=roc_nightly-macos_12_apple_silicon-$DATE-$SHA.tar.gz" >> $GITHUB_ENV  
+        run: echo "RELEASE_TAR_FILENAME=roc_nightly-macos_apple_silicon-$DATE-$SHA.tar.gz" >> $GITHUB_ENV  
```

Comment:
New macOS versions appear to be reverse compatible so we ditch the version number.